config/management/src/genesis.rs

```diff
@@ -91,41 +91,43 @@ impl Genesis {
     }
 
     /// Produces a set of ValidatorRegistration from the remote storage.
+    /// TODO(joshlind): verify that owner account address specified by the validator config matches
+    /// the owner account address that was uploaded to the remote storage. Also verify that the
+    /// operator selection has been signed by the owner (somehow...)
```

Comment:
eeeh, let's not add this verify statement on operator selection, we'll evaluate if it is practical or not